File: task-executor/Driver.java
import java.util.*;
import java.time.*;
import java.math.*;
import java.io.*;

${solution}
public class Driver {
     private static final Scanner scanner = new Scanner(System.in);

    public static void main(String[] args) {
        ${paramsInputSection}
        scanner.close();


        long startTime = System.nanoTime();

        ${returnType} result = new Solution().${methodName}(${paramList});

        long endTime = System.nanoTime();
        long executionTime = endTime - startTime;
        Runtime runtime = Runtime.getRuntime();
        double memory = (double) (runtime.totalMemory() - runtime.freeMemory());
        printResult(result);
        System.out.println(executionTime);
        System.out.println(memory/(1024*1024));
    }
    private static void printResult(int result) {
        System.out.println(result);
    }
    private static void printResult(boolean result) {
        System.out.println(result);
    }
    private static void printResult(String result) {
        System.out.println(result);
    }
    private static void printResult(int[] result) {
        System.out.println(Arrays.toString(result));
    }
    private static void printResult(String[] result) {
        System.out.println(Arrays.toString(result));
    }


    private static String READ_STRING(){
        String input = scanner.nextLine();
        input = input.replace(" ", "");
        return input;
    }

    private static String[] READ_STRING_ARR() {
        String input = scanner.nextLine();
        if (input.equals("[]")) {
            return new String[0];
         }
        // Удаляем квадратные скобки и пробелы
        input = input.replace("[", "").replace("]", "").replace(" ", "");

        // Разделяем строку на элементы
        String[] elements = input.split(",");

        return elements;
    }
    private static int[] READ_INTEGER_ARR() {
        String input = scanner.nextLine();
        if (input.equals("[]")) {
         return new int[0];
         }

        // Удаляем квадратные скобки и пробелы
        input = input.replace("[", "").replace("]", "").replace(" ", "");

        // Разделяем строку на элементы
        String[] elements = input.split(",");

        int[] arr = new int[elements.length];
        for (int i = 0; i < elements.length; i++) {
            arr[i] = Integer.parseInt(elements[i]);
        }

        return arr;
    }
    private static int READ_INTEGER() {
        int number = scanner.nextInt();
        scanner.nextLine();
        return number;
    }
}

File: task-executor/Driver.py
import time
import psutil
from typing import List

${solution}

def READ_STRING():
    return input()

def READ_INTEGER():
    return int(input())

def READ_STRING_ARR():
    input_str = input()
    if input_str == "[]":
        return []
    input_str = input_str.replace("[", "").replace("]", "").replace(" ", "")

    elements = input_str.split(",")

    arr = [str(element) for element in elements]
    return arr

def READ_INTEGER_ARR():
    input_str = input()
    if input_str == "[]":
        return []

    input_str = input_str.replace("[", "").replace("]", "").replace(" ", "")

    elements = input_str.split(",")

    arr = [int(element) for element in elements]
    return arr

if __name__ == '__main__':
${paramsInputSection}
    print(param1)
    print("just string")
    s = Solution()
    start_time = time.time()
    ret = Solution.${methodName}(s, ${paramList})
    end_time = time.time()
    process = psutil.Process()
    memory_info = memory_info = process.memory_info()
    used_memory_kb = memory_info.rss / (1024*1024)
    execution_time = (end_time - start_time) * 1e9
    print(ret)
    print(execution_time)
    print(used_memory_kb)

